11 automated task processing/OrderBooking.WebAPI/MinimalApiConfig.cs

```diff
@@ -0,0 +1,78 @@
+using FastExpressionCompiler;
+using MessageHandler.EventSourcing.DomainModel;
+using Microsoft.Extensions.Azure;
+using OrderAggregate = OrderBooking.OrderBooking;
```

Comment:
From a business analyses perspective I really don't like this rename. The aggregate is representing the Order Booking business capability. It's not an 'Order', which is something that doesn't exist, this is a step that transitions from PurchaseOrder to SalesOrder.

If you want to rename it to avoid the Namespace.Class issue, than I'd suggest to rename it to 'Booking'

Reply:
Entirely agree, yes the reason of renaming is because of VS Code having worried about class and namespace being identical.